deploy/runtime/defaults/Biopower_Single Owner.txt

```diff
@@ -2523,7 +2523,7 @@ const_per_interest_total
 1
 1
 1
-1.82285e+06
+1822850
```

Comment:
Is this format change in the default files normal? There are a lot of instances of this that must have just happened automatically

Reply:
Yeah, that also happened to me on https://github.com/NREL/SAM/pull/421/files and no one thought it was a big deal. I don't know the history, so I'll leave this open until ~4 MT today in case Paul or Ty want to weigh in.